### ISSUE TITLE
Fiddler certificate should be provided in PEM format for Linux

### ISSUE THREAD

**Issue by perlun** [telerik/fiddler-everywhere-docs#65]
(This is more rightfully an issue that should target Fiddler Everywhere than the docs; please point me to the appropriate GitHub issues/JIRA tracker where I can file it if you have a public bug tracker. :smile: We could also resolve it by updating the docs though, so it's not entirely off-topic.)

While trying to get Fiddler Everywhere working on Linux, I realized that the way we currently export the `FiddlerRootCertificate.crt` format is not really optimal for getting applications to trust it. My particular example was Docker, but I also remember having to add it manually to the Firefox trusted certificate store to get it working, so it might be a general problem with this certificate on Debian Buster.

What I did to resolve it was exporting it from Firefox in PEM format (after having imported the `.crt` file provided by Fiddler Everywhere), then replacing the `/usr/share/ca-certificates/extra/FiddlerRootCertificate.crt` file with its content instead of the binary version. Then I followed the steps described in https://docs.telerik.com/fiddler-everywhere/get-started/configuration#configure-on-linux, to update the system certificate store. That way, I could get Docker to trust the Fiddler certificate which was helpful for me in debugging an issue with a private Docker registry etc. (Thanks for a nice product! :+1:)

----

For convenience, here are the PEM version of the Fiddler root certificate as it looks right now; maybe this can help others until the issue is properly resolved upstream.

```pem
-----BEGIN CERTIFICATE-----
MIIDozCCAougAwIBAgIQAJeyjzkHgkQr5NcGvQ+ACTANBgkqhkiG9w0BAQsFADBq
MSswKQYDVQQLDCJDcmVhdGVkIGJ5IGh0dHA6Ly93d3cuZmlkZGxlcjIuY29tMRgw
FgYDVQQKDA9ET19OT1RfVFJVU1RfQkMxITAfBgNVBAMMGERPX05PVF9UUlVTVF9G
aWRkbGVyUm9vdDAeFw0yMDA5MDkwMDAwMDBaFw0zMDA5MTYxNjE2NTVaMGoxKzAp
BgNVBAsMIkNyZWF0ZWQgYnkgaHR0cDovL3d3dy5maWRkbGVyMi5jb20xGDAWBgNV
BAoMD0RPX05PVF9UUlVTVF9CQzEhMB8GA1UEAwwYRE9fTk9UX1RSVVNUX0ZpZGRs
ZXJSb290MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAm798LGRWPDQZ
U0P85V9FJx20QmqXbbi9IzqpLidRd94t+YKWo/aQxlq0pT8jo+P+Z2MPyKMIO7nZ
1N8v5XeTRWRemlvN2IDxsSYbK0Ta2rqf2svDGIR2FhKc/alEpwXwXVhUtdgV5HZh
nBhUNIsfjfjN88w0+yA1wwQlS4bGvh3xoZlgd+7OjuXGgBbH2gAufJ13MUSuZC3k
j0DD9ZwXzUWHpva59nejC5Wz4+tdpntXuyTx3gtyOxjkWfE2HU2clVKGeCtf3uof
06AHSzwUKv11iPNuEJtTMZJMV8BJnFbm0KHzj2w7C4qI7VfgdJUn+QjsOhd//5rF
yRbQdbCCUwIDAQABo0UwQzASBgNVHRMBAf8ECDAGAQH/AgEAMA4GA1UdDwEB/wQE
AwICBDAdBgNVHQ4EFgQUSaKduzG85wOAJiDJqN21En/VGCIwDQYJKoZIhvcNAQEL
BQADggEBAHmUVYDYE5Ezdw6sdfzA5siLD1XFY6Dzcao5Ei1uEqMdHB7RiGMSo1Zp
a3+ApDDjKMnoIG0DYhU92a/o/wlHWwmiJkMta8Or2nsG1vUZf/xv2tJmcAe++tBE
xTV1sOvjo4NhKztFyjYz0RSMB+3GFmxy5Yhr59jbOlKZRVEfRBuHBs+yFdzKcN2t
s0iSxct0hQRmrma4kiYIoU1s27G+WT0a3+J55V1i0Ujeirpj0llxzwXqhQzgzLai
kejvDvecP0UYPJTvzlDDpEqJ1+/qfBYOHsJZQcYG/Jt04Gn5uaMM/ijYZLURyktL
v9ILD4iYtLnkw52m8QUu+szxxLLD6oY=
-----END CERTIFICATE-----
```

**Comment by dawidcrivelli** [comment]
Agreed, there is no documentation on how to export it to the PEM format on linux. This is definitely needed

**Comment by dawidcrivelli** [comment]
Commands to perform the conversion in the shell, using `openssl`
```sh
openssl x509 -inform der -in FiddlerRootCertificate.crt -outform PEM -out FiddlerRootCertificate.pem
```

**Comment by NickIliev** [comment]
Resolved via https://github.com/telerik/fiddler-everywhere-docs/pull/183
